src/IbanNet/Resources.resx

```diff
@@ -1,170 +1,189 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <root>
-	<!-- 
-		Microsoft ResX Schema
+  <!-- 
+    Microsoft ResX Schema 
+    
+    Version 2.0
```

Comment:
Please revert this file. I will check after merge if we can even use newer schema, I remember having issues with the newer code generators with building the .NET Standard 1 targets in some odd cases, f.ex. after adding a new string resource.